xbmc/music/tags/TagLoaderTagLib.cpp

```diff
@@ -462,6 +464,44 @@ bool CTagLoaderTagLib::ParseTag(ID3v2::Tag *id3v2, EmbeddedArt *art, MUSIC_INFO:
           tag.SetUserrating(POPMtoXBMC(popFrame->rating()));
         }
       }
+    else if (it->first == "CHAP")
+    {
+      //We use a chapter number index here to ensure chapters are added in the right order to the map
+      int chapNumber = 0;
+      unsigned int prevTime = 0;
+      for (ID3v2::FrameList::ConstIterator ch = it->second.begin(); ch != it->second.end(); ++ch)
```

Comment:
Could a range based for loop increase readability here? as far as I see you never use `ch` in the for block, except to get the real value out of it at the beginning.

Reply:
It follows the same code structure and usage as the rest of the file.

Reply:
Maybe, and we said there we‘re modernize code as we go. Existing style is secondary. That code might have been written like a decade ago.